server/src/test/java/io/crate/execution/dml/IndexerTest.java

```diff
@@ -671,6 +671,27 @@ public void test_can_index_fulltext_column() throws Exception {
         assertTranslogParses(doc, table);
     }
 
+    @Test
+    public void test_can_index_dedciated_fulltext_to_sub_column() throws Exception {
+        SQLExecutor e = SQLExecutor.of(clusterService)
+            .addTable("""
+                create table tbl (
+                    author object as (
+                        name string
+                    ),
```

Comment:
Important to declare FT separately.

If we declare it together with the column definition, TranslogIndexer -> valueIndexer takes care of handling it properly